optuna/study.py

```diff
@@ -795,6 +795,19 @@ def create_study(
     # type: (...) -> Study
     """Create a new :class:`~optuna.study.Study`.
 
+    Example:
+
+        .. testcode::
+
+            import optuna
+
+            def objective(trial):
+                x = trial.suggest_uniform('x', 0, 10)
```

Comment:
A nit, but we use double quates instead of single quates in the Optuna code (see the [black reference](https://black.readthedocs.io/en/stable/the_black_code_style.html#strings)).
```suggestion
                x = trial.suggest_uniform("x", 0, 10)
```